tools/maybe-trigger-web-snapshot-update.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# maybe-trigger-web-snapshot-update.sh
+#
+# Helper script called at the end of the deploy_api.yml workflow to potentially
+# trigger the update-snapshot.yml workflow in the covid-projections repo to
+# update to the newly generated snapshot.
+
+set -o nounset
+set -o errexit
+
+CMD=$0
+
+# The covid-projections branch to open a PR against.
+BRANCH="develop"
+
+# Checks command-line arguments, sets variables, etc.
+prepare () {
+  # Parse args if specified.
+  if [ $# -ne 3 ]; then
+    exit_with_usage
+  else
+    SNAPSHOT_ID=$1
+    COVID_DATA_MODEL_REF=$2
+    COVID_DATA_PUBLIC_REF=$3
+  fi
+
+  if ! [[ $SNAPSHOT_ID =~ ^[0-9]+$ ]] ; then
+    echo "Error: Specified Snapshot ID ($SNAPSHOT_ID) should be a plain number."
+    echo
+    exit_with_usage
+  fi
+
+  if [[ $COVID_DATA_MODEL_REF != "master" ]] || [[ $COVID_DATA_PUBLIC_REF != "master" ]]; then
```

Comment:
why is this the public ref and the non public ref? i've been a lil confused why the public ref distinction exists as it never worked for me

Reply:
Not 100% sure what you mean by non-public...

These are the git refs of covid-data-model and covid-data-public that were used to build the snapshot.  These are _usually_ "master" but can be changed to point elsewhere when people are manually testing their code in branches, etc.